### PR TITLE
vbc /help should say we support versions up through 14.

### LINKED ISSUE #6692: vbc /help says we support /langversion up to 11, but it should be up to 14.
The output from `vbc /help` says we support `/langversion:`*number* up through *number*=11, but we actually support up through 14. The documentation should be updated.

### PR DESCRIPTION
**Pull request by gafter**
Fixes #6692 

@dotnet/roslyn-compiler Please review this tiny change

### CODE REVIEW

**Comment by TyOverby** [comment]
LGTM

**Comment by gafter** [comment]
@dotnet-bot please test prtest/mac/dbg/unit32

**Comment by gafter** [comment]
@dotnet-bot retest mac/dbg/unit32

**Comment by gafter** [comment]
@dotnet-bot test this please

File: src/Compilers/VisualBasic/Portable/VBResources.resx [hunk 1]
```diff
@@ -5073,7 +5073,7 @@
                                   referenced metadata files. 
                                   import_list:namespace,...
 /langversion:&lt;number&gt;             Specify language version: 
-                                  9|9.0|10|10.0|11|11.0.
+                                  9|9.0|10|10.0|11|11.0|12|12.0|13|13.0|14|14.0
```
Comment (AlekseyTs) [inline src/Compilers/VisualBasic/Portable/VBResources.resx]: It doesn't look like 13 is a valid value:
```
    Public Enum LanguageVersion
        VisualBasic9 = 9
        VisualBasic10 = 10
        VisualBasic11 = 11
        VisualBasic12 = 12
        VisualBasic14 = 14
    End Enum
```

**Comment by AlekseyTs** [comment]
It feels like we should add a unit-test to verify the help text for both compilers.

**Comment by gafter** [comment]
We've specifically avoided adding tests on the C# side that verify the text of resources, as that just doubles the work when changing the resources without adding much value. I'll leave it to you to decide if you want to do it for VB.

**Comment by gafter** [comment]
@dotnet/roslyn-compiler Can I please have a second review of this tiny change?

**Comment by cston** [comment]
LGTM

**Comment by AlekseyTs** [comment]
In my opinion the fact that compiler prints the help should be verified, the presence of the resource provides no guarantees for that.

**Pull request merged**